.github/CODEOWNERS

```diff
@@ -96,10 +96,14 @@ ui/pages/settings/security-tab/profile-sync-toggle      @MetaMask/identity
 
 app/scripts/lib/snap-keyring         @MetaMask/accounts-engineers
 
-# Swaps team to own code for the swaps folder
+# Swaps-Bridge team to own code for the swaps folder
 ui/pages/swaps                                        @MetaMask/swaps-engineers
 app/scripts/controllers/swaps                         @MetaMask/swaps-engineers
 
+# Swaps-Bridge team to own code for the bridge folder
+**/bridge/**                                          @MetaMask/swaps-engineers
```

Comment:
I see we have types, like bridge-status.ts that lives outside of this structure. For files like these, should we:

* colocate them in this or a follow on PR?
* Add specific entries for those files here?

Not blocking, just wanted to call out

Reply:
We can make a new PR to add additional files